src/controller/launch/imu.launch

```diff
@@ -0,0 +1,55 @@
+<!-- Phidgets IMU launch file -->
+
+<launch>
+
+  #### Nodelet manager ######################################################
```

Comment:
Just realizing this now, but comments like these should be enclosed with `<!-- -->`, I've had issues before with using invalid comment characters.

Reply:
This is straight from phidgets website, which makes me think it should be alright